configs/celeryd

```diff
@@ -23,8 +23,8 @@ CELERYD_PID_FILE="/var/run/celery/%n.pid"
 # Workers should run as an unprivileged user.
 #   You need to create this user manually (or you can choose
 #   a user/group combination that already exists (e.g., nobody).
-CELERYD_USER="celery"
-CELERYD_GROUP="celery"
+CELERYD_USER="root"
```

Comment:
This will work in devspace, but it would fail once we get the IDE to production, where change the user to "999" and lock down the filesystem to just that user.

Instead of specifying a user here, can we just run the user as the default by not specifying a user?

Then I think Celery will run as the OS User that started it, which we can control via Kubernetes (like we do for the rest of the IDE)

Reply:
I think Celery will default to the OS user if we don't specify anything here... according to the doc I think?

Reply:
@isaac-taylor oh wow, I didn't know even the root user couldn't access that dir. I'll have to try that, thank you